src/dropin.js

```
@@ -177,6 +178,7 @@ Dropin.prototype._getVaultedPaymentMethods = function (callback) {
       var paymentMethods;
 
       if (err) {
+        console.error('Failed to get vaulted payment methods:', err);
```

I suppose not, but do we want to bubble up the error somehow?